utils/player.py

```diff
@@ -0,0 +1,25 @@
+
+
+class Player():
+
+    def __init__(self, name):
+        self.__name = name
+        self.__curr_pos = 0
+
+    def move_player_by(self, x, board_size):
+        if self.__curr_pos + x <= board_size:
+            self.__curr_pos += x
+
+    def move_player_to(self, x):
+        self.__curr_pos = x
+
+    def is_winner(self, target=100):
```

Comment:
is_winner logic should not be a part of player class. The player should not be allowed to tell if it is a winner.
The GameService or in your case Board should ask the player for their current position and determine if it is a winner.